src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/CourtCaseEventsServiceTest.kt

```diff
@@ -35,9 +33,6 @@ class CourtCaseEventsServiceTest {
   @Mock
   lateinit var offenderService: OffenderService
 
-  @Mock
-  lateinit var pncIdValidator: PNCIdValidator
-
```

Comment:
No need to mock as the CourtCaseService creates its own instance of PNCIdValidator